src/commander/commander.c

```diff
@@ -196,6 +197,17 @@ static commander_error_t _api_insert_remove_sdcard(const InsertRemoveSDCardReque
     return COMMANDER_OK;
 }
 
+static commander_error_t _api_get_root_fingerprint(RootFingerprintResponse* response)
+{
+    char fingerprint[4];
+    bool success = get_root_fingerprint(fingerprint, sizeof(fingerprint));
+    if (!success) {
+        return COMMANDER_ERR_INVALID_STATE;
+    }
+    memcpy(&response->fingerprint, fingerprint, sizeof(fingerprint));
```

Comment:
no need for this, you can use `response->fingerprint` directly above instead of using a local `fingerprint` var.